pkg/registry/servicecatalog/instance/strategy.go

```diff
@@ -108,7 +108,7 @@ func (instanceRESTStrategy) ValidateUpdate(ctx kapi.Context, new, old runtime.Ob
 	if !ok {
 		glog.Warning("received a non-instance object to validate to")
 	}
-	oldInstance := old.(*sc.Instance)
+	oldInstance, ok := old.(*sc.Instance)
```

Comment:
not blocking for this PR, but if `ok` is false, then `oldInstance` will be `nil`